apps/els_core/src/els_config.erl

```diff
@@ -86,33 +86,46 @@
     providers => map()
 }.
 
+-type error_reporting() :: lsp_notification | log.
```

Comment:
As I read through my changes again after some time away from it I came to the conclusion that `no_els_server` and `use_els_server` were kind of bad names for the variants. I think this is more clear.